rocketpool-cli/service/service.go

```diff
@@ -1099,8 +1099,92 @@ func pruneExecutionClient(c *cli.Context) error {
 
 }
 
-// Pause the Rocket Pool service
-func pauseService(c *cli.Context) error {
+// Stops Smartnode stack containers, prunes docker, and restarts the Smartnode stack.
+func resetDocker(c *cli.Context) error {
+
+	fmt.Println("Once cleanup is complete, Rocket Pool will restart automatically.")
+	fmt.Println()
+
+	// Stop...
+	// NOTE: pauseService prompts for confirmation, so we don't need to do it here
+	confirmed, err := pauseService(c)
+	if err != nil {
+		return err
+	} else if !confirmed {
+		// if the user cancelled the pause, then we cancel the rest of the operation here:
+		return nil
+	}
```

Comment:
```suggestion
	}

	if !confirmed {
		// if the user cancelled the pause, then we cancel the rest of the operation here:
		return nil
	}
```

no `else` after conditional return